lib/chai/interface/assert.js

```diff
@@ -41,6 +41,7 @@ module.exports = function (chai, util) {
   };
 
   /**
+   * ### .fail([message])
    * ### .fail(actual, expected, [message], [operator])
```

Comment:
@chaijs/chai Anyone know how the website generation script handles listing multiple function signatures in this way? Or the proper way to do this with jsdoc in general? I struggled with this problem quite a bit in https://github.com/chaijs/check-error/pull/18, and ended up including a single function signature at the top (with optional parameters) but documenting the alternate signatures in the jsdoc body. Not sure what the best approach is.

Reply:
There is a possible solution for handling multiple function signature used in the [`assert.throws`](http://chaijs.com/api/assert/#method_throws) documentation. But I'm not sure this is the _proper_ way of doing though: as a user, this is only by looking at the example section I was able to understand the various signatures for that method.

Reply:
FWIW, I passed the source code through `dox` (which I assume to be the jsdoc parser used for the website). Here is the relevant part of the output:

    "description": {
      "full": "<h3>.fail([message])</h3>\n<h3>.fail(actual, expected, [message], [operator])</h3>\n<p>Throw a failure.</p>",
      "summary": "<h3>.fail([message])</h3>\n<h3>.fail(actual, expected, [message], [operator])</h3>",
      "body": "<p>Throw a failure.</p>"
    },

As you can see, it produces two `<h3>` elements--once for each signature. Which seems quite reasonnable. Maybe we could ping someone at the chai-doc team to check if this is an issue?

Reply:
Yup, chai-docs is using `dox`. That behavior looks promising. @astorije might know for sure.